src/bp/core/app/botpress.ts

```diff
@@ -181,13 +181,22 @@ export class Botpress {
   }
 
   private async maybeStartLocalSTAN() {
+    if (!process.LOADED_MODULES['nlu']) {
+      this.logger.warn(
+        'NLU server is disabled. Enable the NLU module and restart Botpress to start the standalone NLU server'
+      )
+      return
+    }
+
     const config = await this.moduleLoader.configReader.getGlobal('nlu')
 
-    if (!config.nluServer.autoStart) {
+    const autoStart = config.nluServer?.autoStart ?? true
+    if (!autoStart) {
```

Comment:
We should handle the case where endpoint isn't set either. 
Here's what I mean
```ts
if(!autoStart && !config.nluServer.endpoint){
  this.logger.warn('NLU server isn't configured properly, set it to auto start or provide an enpoint')
}
```

Reply:
Agreed, the endpoint is mandatory if it is not autostarted